public/get-results.js

```diff
@@ -0,0 +1,34 @@
+/* global: location, sessionStorage */
+
+(function resultsPage () {
+  var path = location.pathname.slice(1)
+    if (path === "results") {
+      getSurveyResults ()
+    }
+    else return
+  })();
+
+function getSurveyResults () {
+  var insertAnswers = document.getElementById('insert-answers')
+  var surveyType = sessionStorage.getItem('select-survey')
+
+    if (surveyType === 'pre-session') {
+      var preSessionResponses = ['consent', 'today', 'last-week', 'family', 'friends', 'school', 'play', 'next-week']
+      var preSessionAnswers = []
+        for (var i = 0; i < preSessionResponses.length; i++){
+          preSessionAnswers.push(sessionStorage.getItem(preSessionResponses[i]))
+        }
+        insertAnswers.innerHTML = preSessionAnswers
```

Comment:
Best practice to  use
Node.textContent rather than node.innerHTML, inner html is slower, and could potentially open you up to xss attacks down the line